DEPLOYMENT.md

```diff
@@ -5,17 +5,38 @@ on their own so components can be used. It is based on what we've done for
 our testing and production environments, but by no means is this the only
 way to deploy. The general principles and concerns will likely be relevant.
 
-## Dependencies and Environment
+## System Requirements & Dependencies
 
-The following is written with the assumption you are running a Debian-based
-Linux environment. This should work elsewhere, but this is not something
-we have practical experience with. You should have installed:
+The following is written with the assumption you are running a Debian-based Linux distribution (e.g., Ubuntu 22.04, Debian 11).
+Other distributions may work fine, but are not officially tested.
+
+Minimum System Requirements
+OS: Any Linux distribution (Debian-based preferred)
+CPU: At least 4 cores recommended
+RAM: 16GB or more recommended
+Storage: 50GB free, 100GB+ recommended if using a GPU
+GPU: NVIDIA GPU required for certain services
+
+
+IMAGE Server was also successfully deployed on AWS EC2 using the below confiuratings:
+
+Instance Type:	[Preferred] g4dn.xlarge (GPU) or t3.large (CPU-only)
+OS:	Ubuntu 22.04 LTS
+CPU: 4 vCPUs
+RAM: 16GB
+Storage: 1000GB EBS (Elastic Block Store)
+Network: Default VPC with public IPv4
+Security Group: Open ports 22 (SSH), 80 (HTTP), 443 (HTTPS)
+
+## Required Software
+You will need the following installed:
 
 - [Docker Engine](https://docs.docker.com/engine/) and [Docker Compose](https://docs.docker.com/compose/)
     - Note: For version 20.10.13 and later, the `docker-compose-plugin` package provides the `docker compose` subcommand which operates the same was as `docker-compose`.
 - NVIDIA drivers and [nvidia-container-runtime](https://docs.docker.com/config/containers/resource_constraints/#gpu)
     - Many of these containers do not use or do not need to use a GPU, but performance will be greatly reduced.
```

Comment:
I don't know how true this is anymore. We should probably flag ones that work ok on CPU vs GPU